cmd/transportRequestUploadCTS_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"github.com/SAP/jenkins-library/pkg/command"
 	"github.com/SAP/jenkins-library/pkg/mock"
-	"github.com/SAP/jenkins-library/pkg/transportrequest"
+	transportrequest "github.com/SAP/jenkins-library/pkg/transportrequest/cts"
```

Comment:
how about not renaming it to transportrequest? Then we would see the issue codeclimate is pointing out.